lib/collins_auth.rb

```diff
@@ -15,9 +15,12 @@ def self.setup_client(options = {:prompt => false})
 
     def self.load_config(options = {})
       conf = (options[:prompt] == :only) ? {} : (read_config(options[:config_file]) || {}).merge(options)
-
+      # check if host doesn't have a protocol specified
+      if conf.key?(:host) && !conf[:host].nil? && conf[:host].match(/^(http|https)\:\/\/.+/).nil?
```

Comment:
Won't this trigger the error message even if the host just isn't set at all? Also, if `prompt` is `true` maybe we probably want to fall through to the `prompt_creds` call, right?